hell/adventcalendar/2024/6/index.md

```diff
@@ -2,51 +2,69 @@
 title: "aria-labelledby = self"
 layout: layouts/advent.md
 author: "Weston Thayer"
-author_bio: "Short intro about yourself"
+author_bio: "Weston works at Assistiv Labs on new accessibility tools for software teams large and small."
 date: 2024-12-06
 author_links:
-  - label: "xy Blog"
-    url: "#"
-    link_label: "your-awesome-website.com/blog"
-  - label: "yx on ABC"
-    url: "https://abc-social-media-site"
-    link_label: "@xy"
+  - label: "Website"
+    url: "https://assistivlabs.com"
+    link_label: "assistivlabs.com"
 active: true
 intro: "<p>Short description of the post</p>"
```

Comment:
Let me know if this is used anywhere, I can add it.

Reply:
At the bottom of the page, like here: https://htmhell.dev/adventcalendar/2023/2/